src/mixins/filing-template-mixin.ts

```diff
@@ -45,24 +92,24 @@ export default class FilingTemplateMixin extends Vue {
       filing: {
         header: {
           name: INCORPORATION_APPLICATION,
-          certifiedBy: this.stateModel.certifyState.certifiedBy,
-          email: this.stateModel.defineCompanyStep.businessContact.email,
+          certifiedBy: '',
+          email: '',
           date: this.stateModel.currentDate
         },
+        business: {
+          legalType: EntityTypes.BCOMP,
```

Comment:
This should come from the IA.

Having this default value concerns me. I think we should not default it and throw an error if the IA doesn't contain this value.

Reply:
got rid of this method any way, so should be better